content/applications/productivity/iot/config/updating_iot.rst

```diff
@@ -0,0 +1,145 @@
+==============
+Updating (IoT)
+==============
+
+Due to the complexity of the :abbr:`IoT (Internet of Things)` box, and virtual Windows :abbr:`IoT
+(Internet of Things)` box, the term 'updating' can mean several different things.
+
+The actual drivers can be updated, the core code on the :abbr:`IoT (Internet of Things)` box can be
+updated, and a new image can be flashed (physical :abbr:`IoT (Internet of Things)` box).
+
+This document explores the various ways to update :abbr:`IoT (Internet of Things)` boxes to ensure
+smooth operation of :abbr:`IoT (Internet of Things)` box processes and devices.
+
+.. _iot/config/flash:
+
+Flashing the SD card on IoT box
+===============================
+
+.. important::
+   This update does **not** apply to the Windows :abbr:`IoT (Internet of Things)` box (Odoo 16 and
+   higher).
+
+   To update the Windows :abbr:`IoT (Internet of Things)`, first, uninstall the previous version of
+   the Odoo Windows program, and then reinstall it using the most up-to-date installation package.
+
+   To begin the installation, navigate to the Odoo 16 (or higher) installation package for
+   Enterprise or Community - Windows edition, at `Odoo's download page
+   <https://odoo.com/download>`_.
+
+In some circumstances, the :abbr:`IoT (Internet of Things)` box's micro SD Card may need to be
+re-flashed with *Etcher* software to benefit from Odoo's latest :abbr:`IoT (Internet of Things)`
+image update.
+
+This means the Odoo :abbr:`IoT (Internet of Things)` box software may need to be updated in
+instances of a new :abbr:`IoT (Internet of Things)` box, or when a handler's update, and an update
+from the :abbr:`IoT (Internet of Things)` box home page, does not fix issues.
+
+.. warning::
+   It is often imperative to re-flash the :abbr:`IoT (Internet of Things)` box's image after
+   upgrading the Odoo database to a new version.
+
+.. note::
+   A computer with a micro SD card reader/adapter is **required** to re-flash the micro SD card.
+
+Navigate to Balena's website, and download `Etcher <https://www.balena.io/>`_. It is a free,
+open-source utility, used for burning image files onto drives.
+
+Click to `download <https://www.balena.io/etcher#download-etcher>`_. Install and launch the program
+on the computer.
```

Comment:
Indicate that we're getting into the steps now with transitional language, since the first few blocks of text prior to this followed a non-linear order and were mostly theoretical.

Suggested rewrite:
```suggestion
First, begin by downloading `Etcher <https://www.balena.io/etcher#download-etcher`_. It is a free,
open-source utility, used for burning image files onto drives. After the download completes, install
and launch the program on the computer.
```